service/worker/migration/activities.go

```diff
@@ -482,3 +541,197 @@ func (a *activities) SeedReplicationQueueWithUserDataEntries(ctx context.Context
 		activity.RecordHeartbeat(ctx, heartbeatDetails)
 	}
 }
+
+func (a *activities) createReplicationTasks(ctx context.Context, request *genearteAndVerifyReplicationTasksRequest, detail *replicationTasksHeartbeatDetails) error {
+	rateLimiter := quotas.NewRateLimiter(request.RPS, int(math.Ceil(request.RPS)))
+
+	for i := 0; i < len(request.Executions); i++ {
+		r := &detail.Results[i]
+		if r.isCompleted() {
+			continue
+		}
+
+		we := request.Executions[i]
+		tags := []tag.Tag{tag.WorkflowType(forceReplicationWorkflowName), tag.WorkflowNamespaceID(request.NamespaceID), tag.WorkflowID(we.WorkflowId), tag.WorkflowRunID(we.RunId)}
+
+		resp, err := a.historyClient.DescribeMutableState(ctx, &historyservice.DescribeMutableStateRequest{
+			NamespaceId: request.NamespaceID,
+			Execution:   &we,
+		})
+
+		switch err.(type) {
+		case nil:
+			if resp.GetDatabaseMutableState().GetExecutionState().GetState() == enumsspb.WORKFLOW_EXECUTION_STATE_ZOMBIE {
+				a.forceReplicationMetricsHandler.Counter(metrics.EncounterZombieWorkflowCount.GetMetricName()).Record(1)
+				a.logger.Info("createReplicationTasks skip Zombie workflow", tags...)
+
+				r.Status = VERIFY_SKIPPED
```

Comment:
Seems like it is not just skip verify, it also skips generate replication task. In addition, should we filter this in the force replication API?

Reply:
yes. are u suggesting that we should not skips generate replication task for Zombie WF? 
what does "should we filter this in the force replication API?" mean?

Reply:
synced offline. Not do this at current stage.